src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
Override equals and hashcode, cause it's very important when you work with `Set`